spec/features/state_file/editing_df_xml_spec.rb

```diff
@@ -27,7 +27,7 @@
 
     xml_before = StateFileNyIntake.last.raw_direct_file_data.strip
 
-    expect(page).to have_text I18n.t('state_file.questions.name_dob.edit.title1')
+    expect(page).to have_text I18n.t('state_file.questions.data_review.edit.title')
```

Comment:
Since this test is also ensure we can edit the DF xml, this test should bring back the check for the federal info controller similar to how it is here: https://github.com/codeforamerica/vita-min/pull/4155/files#diff-904908460ac030efc6510fd69668d5a2c9fa2e5438f5d885872601e28f420815

I don't know how much coverage we should have on the federal info form, but it did break silently in the past, so we should consider adding increased coverage this year

Reply:
Done - I'm not really doing much in the federal info controller (Since it is still skipped over in non dev contexts), but we make sure it does load and redirect into the NameDobController